platform/ext/target/nordic_nrf/common/core/target_cfg.c

```diff
@@ -46,6 +46,59 @@
 #define PIN_XL1 0
 #define PIN_XL2 1
 
+/* Use the SPU_PERIPH_PERM_SECUREMAPPING_NonSecure symbol to determine
+ * if we have and old or new SPU.
+ */
+#ifdef SPU_PERIPH_PERM_SECUREMAPPING_NonSecure
+/* TODO: Remove this temporary fix when the patch from NRFX-3570 is available.
```

Comment:
I know how bad our memory is so maybe add a Jira to this TODO :)

Reply:
we'll get a multiple definition error when NRFX is updated so it shouldn't be necessary to have a reminder :)